addons/website_forum/views/forum_forum_templates_post.xml

```diff
@@ -165,7 +165,7 @@
     <div t-if="question.tag_ids" class="o_wforum_index_entry_tags ms-0">
         <a  t-foreach="question.tag_ids" t-as="question_tag"
             t-attf-href="/forum/#{slug(question_tag.forum_id)}/tag/#{slug(question_tag)}/questions?#{keep_query(filters='tag')}"
-            t-attf-class="badge text-bg-secondary #{'ms-1' if not question_tag_first else ''} fw-normal"
+            t-attf-class="badge text-bg-secondary #{'ms-1' if not question_tag_first else ''} fw-normal o_tag o_tag_color_#{question_tag.color}"
```

Comment:
I am a bit afraid of this class, it is apparently in core/tags_list but I am wondering if this is not a mistake/chance that this is in assets_frontend. I see we also have `o_website_tag` which was introduced for the "website_controller_page" feature... And probably some more for events / eLearning? I would not mind having @stefanorigano 's opinion on this.

Reply:
Thanks for the heads-up! 👍

Yes, we've got a bunch of different tag classes floating around. We're actually working on cleaning this up and making badges consistent across all modules, `s_badge` included (see WIP pr https://github.com/odoo/odoo/pull/162935).

About using `o_tag` in the frontend - I don't see any "risks". Even if we tweak backend colors, for example, the correlation between frontend and backend tags stays intact. After all, you're assigning these tags from the backend and it's what you expect.
That said, `o_website_tag` would be a better fit. These classes seems generated the same way, but at least are defined in the website module, which makes more sense for frontend.

cc. @chgo-odoo

Reply:
> That said, o_website_tag would be a better fit. These classes seems generated the same way, but at least are defined in the website module, which makes more sense for frontend.

Yeah... but it's in the website controller pages file, which is a feature which is doomed to die or be heavily refactored by us at some point ^^ Anyways, let's use that :+1: If we ever remove them, we'll need to write upgrade script / compatibility code either way.

@panv-odoo Let's use `o_website_tag` instead of `o_tag` (make sure it has an equivalent desired effect).